python/tests/test_topology.py

```diff
@@ -917,44 +917,50 @@ def ts_kc_distance_incremental(ts1, ts2, lambda_=0):
     t1_vecs = KCVectors(ts1.num_samples)
     t2_vecs = KCVectors(ts2.num_samples)
 
+    t1_clean_nodes = np.zeros(ts1.num_nodes, dtype=bool)
```

Comment:
``clean`` isn't quite the right word I think. What we're marking is unaffected subtrees, isn't it? But the usage here is the opposite, where we're traversing into the clean subtrees.

It would be good to think the terminology through again, and add a few comments there to make it clear what's happening. (I think this is going to be a recurring pattern in top-down incremental algs, so I want to make sure we get it right first time.)